srml/system/src/lib.rs

```diff
@@ -364,6 +364,10 @@ pub fn ensure_inherent<OuterOrigin, AccountId>(o: OuterOrigin) -> Result<(), &'s
 }
 
 impl<T: Trait> Module<T> {
+	pub fn get_account_nonce(account: &T::AccountId) -> T::Index {
```

Comment:
this is exactly equivalent to `account_nonce` function (`pub AccountNonce get(account_nonce)` creates it)